hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report_views.xml

```diff
@@ -65,10 +65,17 @@
               action="hr_attendance.hr_attendance_action_graph"
     />
 
-    <menuitem id="menu_hr_attendance_theoretical_report"
+    <menuitem id="menu_hr_attendance_theoretical_label"
```

Comment:
Maybe you should keep the id for one of the menu items, probably on the report and rename this as menu_hr_attendance_theoretical_root or something similar.

Reply:
Okay, already changed it and reverted the commas change!